src/Chapter3.hs

```diff
@@ -907,9 +1005,23 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+
+newtype Gold = Gold Integer
+  deriving (Show)
+
 class Append a where
     append :: a -> a -> a
 
+instance Append Gold where
+    append (Gold x) (Gold y) = Gold (x + y)
+
+instance Append [a] where
+    append x y = x ++ y
```

Comment:
If you like, you can even eta-reduce this implementation and make it even shorter :slightly_smiling_face: 
```suggestion
    append = (++)
```